roles/wordpress-install/tasks/main.yml

```diff
@@ -37,6 +37,27 @@
   when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
   changed_when: "'WordPress is already installed.' not in wp_install_results.stdout"
 
+- name: Check Existing Permalink Structure
+  command: wp rewrite list --format=count
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_permalink_check_results
+  with_dict: wordpress_sites
+  when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
+  changed_when: False
+
+- name: Setup Permalink Structure
+  command: wp rewrite structure {{ item.value.permalink_structure }}
```

Comment:
Can we set a default here?

`item.value.permalink_structure  | default("/%postname%/")`